### ISSUE TITLE
Is it possible to share how to labels for training ECG fine-tune model?

### ISSUE THREAD

**Issue by AleyNg443** [harryjdavies/HeartGPT#3]
I’m curious about the process of labeling the fine-tune training dataset for Atrial Fibrillation and PPG Beat detection. Is it possible to share how it's done?

**Comment by zi12138** [comment]
I have the same question, could you please answer it
